internal/util/importutil/import_wrapper.go

```diff
@@ -27,7 +44,21 @@ import (
 const (
 	JSONFileExt  = ".json"
 	NumpyFileExt = ".npy"
-	MaxFileSize  = 1 * 1024 * 1024 * 1024 // maximum size of each file
+
+	// this limitation is to avoid this OOM risk:
+	// for column-based file, we read all its data into memory, if user input a large file, the read() method may
+	// cost extra memory and lear to OOM.
+	MaxFileSize = 1 * 1024 * 1024 * 1024 // 1GB
+
+	// this limitation is to avoid this OOM risk:
+	// simetimes system segment max size is a large number, a single segment fields data might cause OOM.
+	// flush the segment when its data reach this limitation, let the compaction to compact it later.
+	MaxSegmentSizeInMemory = 512 * 1024 * 1024 // 512MB
+
+	// this limitation is to avoid this OOM risk:
+	// if the shard number is a large number, although single segment size is small, but there are lot of in-memory segments,
+	// the total memory size might cause OOM.
+	MaxTotalSizeInMemory = 2 * 1024 * 1024 * 1024 // 2GB
```

Comment:
IIUC, maxTotalSize should be equal to the file size limit on a Bulk Load task, which is 1GigBytes.

Why are we setting it 2GigBytes here while loading backup files?

Reply:
No, for column-based task, there are several files, each file limit is 1GB, but the total size may exceed 1GB